businessServer/client/components/SingleForm.js

```diff
@@ -96,24 +98,23 @@ const SingleForm = React.createClass({
     // console.log(this.props.data.topLevelNodeIndex)
     // console.log(this.props.topLevelNodes[this.props.prodSelected][this.props.data.topLevelNodeIndex+1]);
     var nextTreePointer;
-    // if(this.props.topLevelNodeIndex < this.props.topLevelNodes[this.props.prodSelected].length-1){
-    //   nextTreePointer = this.props.topLevelNodes[this.props.prodSelected][this.props.data.topLevelNodeIndex+1];
-    // }
-    // else{
-    //   nextTreePointer = null;
-    // }
+    if(this.props.topLevelNodeIndex < this.props.topLevelNodes[this.props.prodSelected].length-1){
+      nextTreePointer = this.props.topLevelNodes[this.props.prodSelected][this.props.data.topLevelNodeIndex+1];
+    }
+    else{
+      nextTreePointer = null;
+    }
```

Comment:
leave this code commented out.  101-106.
the nextTreePointer is assigned in 109-114.